sickbeard/postProcessor.py

```diff
@@ -957,6 +958,15 @@ def process(self):
                 else:
                     cur_ep.release_name = ""
 
+                if cur_ep.torrent_hash != '':
+                    client = clients.getClientIstance(sickbeard.TORRENT_METHOD)()
```

Comment:
i don't remember. probably i have get this peace of code from other part of sickrage....